wct.conf.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  verbose: true,
```

Comment:
@devnook do you mind if we disable `verbose` by default? it's so much output... you can still enable it very easily with `-v` option anyway.